src/cli/cli.cpp

```diff
@@ -5279,6 +5315,20 @@ template <> otError Interpreter::Process<Cmd("networkname")>(Arg aArgs[])
     {
         OutputLine("%s", otThreadGetNetworkName(GetInstancePtr()));
     }
+    /**
+     * @cli networkname (name)
+     * @code
+     * networkname OpenThread
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otThreadSetNetworkName
+     * @cparam networkname @ca{name}
+     * @par
+     * Set the Thread Network Name.
+     * @par
+     * NOTE: The current commissioning credential becomes stale after changing this value. Use pskc to reset.
```

Comment:
Can you make this not all uppercase

Note: 

Also pskc should be in code font `pskc`